docs/changelog.md

```diff
@@ -9,6 +9,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/plugins/dependencies/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/.config.yml)
 
+* **feature-toolbar**
+  * 🆕 Added a new parameter to `ToolbarFeature` which allows clients to specify if the search terms should be shown when the toolbar is in display mode.
```

Comment:
```suggestion
  * 🆕 Added a new parameter `shouldDisplaySearchTerms` to `ToolbarFeature` which allows clients to specify if the search terms instead of the URL should be shown when the toolbar is in display mode.
```

Reply:
Found myself using the `[Bug #ticket_number]` format too. Opened [1811801](https://bugzilla.mozilla.org/show_bug.cgi?id=1811801) in this scope.